values.yaml

```diff
@@ -862,6 +862,21 @@ connectInject:
     proxyDefaults: |
       {}
 
+  sidecarProxy:
+    # Set default resources for sidecar proxy. If null, that resource won't
+    # be set.
```

Comment:
Should this be on the line above? Also, should we mention here that they could be overwritten via an annotation?

Reply:
I put the docs here because sidecarProxy is a top-level setting that we might put other settings into. Yes, will mention the ability to override.